lang/ja.js

```diff
@@ -26,13 +26,13 @@ export default {
 	"components.form-element.defaultError": "{label} は無効です。",
 	"components.form-element.defaultFieldLabel": "フィールド",
 	"components.form-element.input.email.typeMismatch": "電子メールが無効です",
-	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
-	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {数値は {max} より小さくなければなりません。} other {数値は {max} 以下でなければなりません。}}",
-	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {数値は {min} より大きくなければなりません。} other {数値は {min} 以上でなければなりません。}}",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {数値は {min} より大きく {max} 未満である必要があります。} other {数値は {min} より大きく {max} 以下である必要があります。}}} other {{maxExclusive, select, true {数値は {min} 以上 {max} 未満である必要があります。} other {数値は {min} 以上 {max} 以下である必要があります。}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {数値は {max} 未満である必要があります。} other {数値は {max} 以下である必要があります。}",
+	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {数値は {min} より大きい必要があります。} other {数値は {min} 以上である必要があります。}",
 	"components.form-element.input.text.tooShort": "{label} は {minlength} 文字以上である必要があります",
 	"components.form-element.input.url.typeMismatch": "URL が有効ではありません",
 	"components.form-element.valueMissing": "{label} は必須です。",
-	"components.form-error-summary.errorSummary": "{count, plural, other {送信した情報に {count} 件のエラー が見つかりました}}",
+	"components.form-error-summary.errorSummary": "{count, plural, one {送信した情報に 1 件のエラーが見つかりました} other {送信した情報に {count} 件のエラーが見つかりました}}",
```

Comment:
Before `{Count} errors found in the information submitted` after is the same

Reply:
Any thoughts on why this `plural-key  components.form-error-summary.errorSummary` error could be happening? The format looks consistent with `en` to me.

Reply:
It's because Asian languages shouldn't be using "one". From [the docs](https://formatjs.io/docs/core-concepts/icu-syntax#plural-format):
> This category is used for languages that have grammar specialized specifically for one item. Many languages, but not all, use this plural category. (Many popular Asian languages, such as Chinese and Japanese, do not use this category.)